applications/CETF.md

```diff
@@ -0,0 +1,246 @@
+- **Team Name:** CETF
+- **EOS Payment Address:** cet.f
+- **[Level](https://github.com/eosnetworkfoundation/grant-framework#grant-levels):** 1
+
+## Project Overview
+
+"New Proposal"
+
+### Overview
+
+- **Name:** CETF - Crypto Exchange Traded Funds
+- **Brief Description:** There are thousands of crypto assets on the market, it requires a lot of research to know which assets to invest in. Most people do not know and don’t want to know anything about whitepapers, tokenomics, or on-chain analytics. Yet, they would like to invest in crypto. One of the solutions are ETFs/indices, we are building decentralized asset management protocol that enables to create and manage such ETFs. In other words, our protocol enables to put many tokens into one token. Through simplistic UX the token is available to be acquired by investors without prior exposure to crypto.
+- **Relationship to EOSIO:** Our market research has shown that each chain has their own asset management protocol, our aim is provide a solution for EOSIO.
+- **Reason for Interest:** We've have deep interest in finance and we've been building on EOSIO for more than two years. Fundamental interest is to maximize number of individuals who have financial freedom.
+- **Pitch deck:** https://drive.google.com/file/d/1FldnAjAXED5Knl_Ud76PRCk8wx5UIjAW/view?usp=sharing
+- **Helios interview:** https://youtu.be/JFsF0z1yz1Y
+
+
+
+### Project Details
+
+We created first ETF on EOSIO called EOSETF, it consists of 13 EOS mainnet tokens. We deployed the fund to the mainnet in 2021 Q3. UI can accessed via https://eosetf.io/, contracts deployed to https://bloks.io/account/cet.f. Currently there are more than 150 investors holding EOSETF, TVL has peaked over 35k USD. The price of the EOSETF has been pegged to the price of the underlying tokens it consists of, since some users have created arbitrage bots to capture price discrepancies, and others just perfrom the arbitrage manually. At current stage (Phase 1) EOSETF is a proof of concept/MVP. We saw enough traction to start building the Phase 2.
+
+Phase 2 is being tested here: https://testing.eosetf.io/.
+
+New features in Phase 2:
+
+1. Automatic rebalancing
+
+In current Phase 1 it is manual and laborious process to rebalance the fund. The new rebalancing function enables easily to include and exclude tokens from the fund. EOSETF (and the future funds) will have fund managers that through https://polling.eosetf.io/ (currently connected to the testing fund, which you can try to rebalance. Pick tokens you'd like to include or exclude, click VOTE and then REBALANCE). There is no limit as to how many fund managers can participate in the rebalancing. The votes of fund managers are summed together to produce the final allocation. We've on-boarded three individuals from EOS community who have the skill and are interested in managing the EOSETF. Based on the votes of fund managers the REBALANCING function automatically exectues the trades via https://defibox.io/.
```

Comment:
Will the open source software allow independent funds to be created with different sets of initial fund managers?

How can the fund managers be changed? Is there an owner of the fund who has authority to change the fund managers and their voting weights?

Reply:
Can you comment on the decision behind having the contract rebalance by executing actions against a specific on-chain DEX versus the alternative of creating arbitrage incentives (with respect to any viable exchange) for users to interact with an automated market maker (AMM) built into the contract in a way that pushes the balances towards the intended target?

Also, the interactions with the DEX will incur an expense to the contract. I'm assuming that this expense is taken from pooled revenue generated by the contract before the remaining profit is somehow distributed to the appropriate parties. And I assume the revenue is generated from fees charged to those holding the funds, correct? Can you comment a little on how these fees are charged? Also, how are their amounts changed and in particular how are they adapted to ensure profitability and continued rebalancing despite changing DEX fees?